src/FirstSteps/RPG/Equipment.cs

```diff
@@ -0,0 +1,78 @@
+﻿using FirstSteps.RPG.Items;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FirstSteps.RPG.Heroes;
+
+namespace FirstSteps.RPG
+{
+    public class Equipment
+    {
+        private List<Item> backPack = new List<Item>();
+        const int maxBackpack = 5;
+        const int _maxWeight = 40;
+        public int _coinsBag;
+        private Hero _hero;
```

Comment:
`private Hero _hero;` no no no. Hero has his own equipment, not the other way around.